components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -2973,4 +2974,7 @@ public static class ApiManagerConfigurationConstants {
     public static final String API_TEST_KEY_CONFIGURATIONS = "APITestKeyConfigs";
     public static final String API_TEST_KEY_VALIDITY_EXP = "Exp";
     public static final String DEFAULT_API_TEST_KEY_VALIDITY = "600";
+
+    public static final String DEVELOPMENT_ENV = "DEVELOPMENT";
+    public static final String PRODUCTION_ENV = "PRODUCTION";
```

Comment:
Shall we use the same auds that we use in our OAuth2 tokens?
https://github.com/wso2-enterprise/choreo-apimgt-extensions/blob/a560e1fb7da27eacc549c73154a9afd32eb92397/components/org.wso2.am.choreo.extensions.keymanager.asgardeo/src/main/java/org/wso2/am/choreo/extensions/keymanager/asgardeo/AsgardeoKMConstants.java#L41-L42